docs/schema.md

```diff
@@ -577,6 +581,10 @@ properties:
             verbs: ['*']
```

Comment:
nit (above): `apiGroups` entries shouldn't include the version. In the above examples, they should be `apps.kubernetes.io` and `etcd.database.coreos.com`.

Reply:
Updated.